src/data_review_tool/pages/article_review.py

```diff
@@ -373,19 +484,25 @@ def update_button(checked):
             ),
         ]
 
-# Add home button callback
 @callback(
     Output("location_home", "href"),
     Input("home-button", "n_clicks"),
     prevent_initial_call=True,
 )
 def cell_clicked(n_clicks):
+    """Return the href of the home button to redirect to the home page
+    
+    Args:
+        n_clicks (int): The number of times the home button has been clicked
+    
+    Returns:
+        str: The href of the home button
+    """
     if n_clicks:
         return f"http://127.0.0.1:8050/"
```

Comment:
bugfix: The IP address should be replaced with `0.0.0.0` so that users can access an instance of the tool running on a VM.